src/validation/movieValidation.js

```diff
@@ -0,0 +1,18 @@
+import { MIN_MOVIE_RATING, MAX_MOVIE_RATING } from '../constants';
+
+export const isValidString = (value) => {
+  if (typeof value === 'string' && value.trim() !== '') return true;
+
+  return false;
+};
+
+export const isValidRating = (value) => {
+  if (
+    typeof value === 'number' &&
+    MIN_MOVIE_RATING <= value &&
+    value <= MAX_MOVIE_RATING
+  )
+    return true;
+
+  return false;
+};
```

Comment:
```suggestion
export const isValidString = (value) => {
  return typeof value === 'string' && value.trim() !== '';
};

export const isValidRating = (value) => {
  return (
    typeof value === 'number' &&
    MIN_MOVIE_RATING <= value &&
    value <= MAX_MOVIE_RATING
  );
};
```
이렇게 표현해도 무방할 것 같네요!?